backend/.env.example

```
@@ -7,6 +7,11 @@ NODE_ENV=development
 # Frontend URL for CORS (use comma-separated for multiple origins)
 CORS_ORIGIN=http://localhost:3000
 
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)